sklearn/multioutput.py

```diff
@@ -565,18 +565,17 @@ class ClassifierChain(MetaEstimatorMixin, ClassifierMixin, _BaseChain):
 
         If order is 'random' a random ordering will be used.
 
-    cv : int, cross-validation generator or an iterable, optional \
-    (default=None)
+    cv : int, cross-validation generator or an iterable, default=None
         Determines whether to use cross validated predictions or true
         labels for the results of previous estimators in the chain.
-        If cv is None the true labels are used when fitting. Otherwise
+        If None, the true labels are used when fitting. Otherwise
         possible inputs for cv are:
 
         - integer, to specify the number of folds in a (Stratified)KFold,
         - :term:`CV splitter`,
         - An iterable yielding (train, test) splits as arrays of indices.
 
-    random_state : int, RandomState instance or None, optional (default=None)
+    random_state : int, RandomState instance, default=None
```

Comment:
Since we have not made a decision on including `None` or not: https://github.com/scikit-learn/scikit-learn/issues/17748 , I would suggest revert the changes to `random_state`.

Reply:
Thank you for the comment @thomasjpfan . I will revert the changes to random_state and commit. Should I also revert changes to n_jobs?

Reply:
> Should I also revert changes to n_jobs?

Yea, since it is unclear what direction we want to go in.

Reply:
I have now reverted changes to random_state and n_jobs.